Added note on path error (#9656)
MicrosoftDocs/sql-docs: docs/t-sql/statements/create-server-audit-transact-sql.md

#### FILEPATH = '*os_file_path*'

> The path of the audit log. The file name is generated based on the audit name and audit GUID. This path must be valid or the audit cannot be created.

Reviewer: ```suggestion
The path of the audit log. The file name is generated based on the audit name and audit GUID. If this path is invalid, the audit isn't created.
```
